ynput/ayon-equalizer
# Plate handles, frame rate and offset

## Changelog Description
Plate that has non-zero handles is loaded with handles, offset is set to first frame, frame rate is set.

## Additional info
Might close https://github.com/ynput/ayon-equalizer/issues/9

## Testing notes:

1. Load plate with non-zero handles
2. Check camera attributes
![image](https://github.com/user-attachments/assets/6ef939b3-4808-451b-b8d4-75d64cba1c0f)
